csm/conf/uds.py

```diff
@@ -72,15 +72,13 @@ def remove_csm_config(cls):
     @staticmethod
     def generate_haproxy_frontend_config():
         cluster_ip = SaltWrappers.get_salt_call('pillar.get', 'cluster:cluster_ip')
-        mgmt_vip = SaltWrappers.get_salt_call('pillar.get', 'cluster:mgmt_vip')
         return f"""\
 frontend uds-frontend
     mode tcp
     option tcplog
     bind 127.0.0.1:5000
     bind ::1:5000
     bind {cluster_ip}:5000
-    bind {mgmt_vip}:5000
```

Comment:
I am fine if mgmt_vip is not used so as to bypass the issue for active-passive csm. I could not come up with any other simple alternative. Using transparent is a hack which will need sufficient testing.